core/src/main/java/io/questdb/griffin/SqlUtil.java

```diff
@@ -846,6 +883,14 @@ private static long implicitCastStrVarcharAsTimestamp0(CharSequence value, int c
         return Numbers.LONG_NULL;
     }
 
+    private static boolean isNotWrapperModel(QueryModel model) {
```

Comment:
nit: shall we call this method `isNotPlainSelectModel` since we already have `isPlainSelect` method?